### DISCUSSION TITLE
Meet the Developers

### DISCUSSION THREAD

**Discussion by jsimonclark** [BattMoTeam/BattMo#11] [General]
Hello everyone! My name is Simon Clark. I am a Research Scientist at SINTEF based in Trondheim, Norway. 

My journey to battery modelling started in outer space! I originally worked as a FEM simulation engineer on ESA missions like Solar Orbiter. Around 2012, I started applying those skills towards modelling of electrochemical systems and completed my Dr. rer. nat. degree in Computational Electrochemistry at the German Aerospace Center in the group of Prof. Dr. Arnulf Latz. 

Since joining SINTEF, my work is focused on trying to make battery modelling tools more powerful and more accessible for the people who need to use them.

I hope you enjoy working with BattMo!

**Comment by augustjohansson** [comment]
Hi! I'm August Johansson, and I am a Research Scientist at the Computational Geosciences group at SINTEF, Oslo, Norway. My background and training is in finite element methods. I did my PhD in Umeå, Sweden with Mats Larson, a Miller Fellowship at UC Berkeley with James Sethian, and then a postdoc at Simula in Oslo with Anders Logg, where I worked on the FEniCS project. At SINTEF, I have mainly worked with MRST, the Matlab Reservoir Simulation Toolbox, which is the foundation of BattMo. 

I enjoy working with electrochemical systems because there's great potential (haha) from both a numerical, research-wise perspective, as well as an industrial and societal perspective.
